index.html

```diff
@@ -1,57 +1,163 @@
+          <li>CANOPY</li>
+          <li>BACKEND DEV</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="flex project_coding_langs">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="preview_button outlinings">See Project</button>
+      </article>
+      <article class="card flex white-back">
+        <img src="./images/project_3.png" alt="tonic project preview">
+        <h3>Tonic</h3>
+        <ul class="flex project_details">
+          <li>CANOPY</li>
+          <li>BACKEND DEV</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="flex project_coding_langs">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="preview_button outlinings">See Project</button>
+      </article>
+      <article class="card flex white-back">
+        <img src="./images/project_4.png" alt="Multi-Post project preview">
+        <h3>Multi-Post Stories</h3>
+        <ul class="flex project_details">
+          <li>CANOPY</li>
+          <li>BACKEND DEV</li>
+          <li>2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <ul class="flex project_coding_langs">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="preview_button outlinings">See Project</button>
+      </article>
+    </section>
+    <section class="about white-back">
+      <article>
+        <h1>About <br> Myself</h1>
+        <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my
+          work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <div class="socials">
+          <h6>LET'S CONNECT</h6>
+          <ul class="icons_social__container flex">
+            <li>
+              <a href="#"><i class="fa-brands fa-twitter"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-linkedin-in"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-github"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+            </li>
+            <li>
+              <a href="#"><i class="fa-brands fa-instagram"></i></a>
+            </li>
+          </ul>
+        </div>
+        <button class="outlinings">Get My Resume</button>
+        <ul class="about__stacks">
+          <li class="about_stacks__item">
+            <span>Languages</span>
+            <i class="fa-solid fa-chevron-right"></i>
+          </li>
```

Comment:
- [ ] As requested by the [previous reviewer](https://github.com/UREYPRICE/Microverse-Portfolio-Mobile-First/pull/4#discussion_r979230780), kindly make sure that you include the individual languages as it is provided in the Figma design template you chose. 

**This is how your design currently looks like:**

![image](https://user-images.githubusercontent.com/86318284/192101818-75447b87-8a0e-4137-a5c5-a72c8c26e8cb.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/192101849-1cbc8598-dd3a-4f77-b576-b849dd0e28d7.png)

- [ ] I have also seen you mention in your pull request description that the languages icons are not included in the project. I find this very strange because I can clearly see them and I can even export them and use them as PNG, JPG SVG or as  PDF. I will still require you to include them. Below is a screenshot of the Figma section.

![image](https://user-images.githubusercontent.com/86318284/192101766-5dc156ec-a375-4004-98bd-aced353b868e.png)

**This demonstrated as possible, kindly make sure you include them.**

Reply:
can you kidnly look at it now?